packages/paste-core/components/alert-dialog/src/AlertDialogFooter.tsx

```diff
@@ -12,11 +12,21 @@ export interface AlertDialogFooterProps extends HTMLPasteProps<'div'>, Pick<BoxP
   onConfirmLabel: string;
   onDismiss: () => void;
   onDismissLabel: string;
+  disableDestructive?: boolean;
```

Comment:
I can definitely change to `disabled` I chose to only disable the destructive button because that's the only need we have. As far as I know there's no need for anyone to disable the default alert dialog confirmation button, and the ticket only required disabling destructive.

Reply:
update: changing to onConfirmDisabled